sources/packages/backend/package.json

```diff
@@ -23,9 +23,16 @@
     "test:e2e": "cross-env ENVIRONMENT=test jest --config ./apps/api/test/jest-e2e.json",
     "migration:run": "node --abort-on-uncaught-exception --require ts-node/register apps/db-migrations/src/main.ts",
     "migration:create": "cd apps/db-migrations/src/migrations && npx typeorm-ts-node-commonjs migration:create",
-    "migration:revert": "npx typeorm-ts-node-commonjs migration:revert -d apps/db-migrations/src/data-source.ts"
+    "migration:revert": "npx typeorm-ts-node-commonjs migration:revert -d apps/db-migrations/src/data-source.ts",
+    "db:seed:test": "npm run build test-db-seeding && npm run migration:run &&  node dist/apps/test-db-seeding/main.js",
+    "//db:seed:test:clean": "Drops the database and all its data. Be careful with this command on production since this command will erase all your database tables and their data.",
```

Comment:
Are you going to keep both comments?

Reply:
I kept both to know the dev's thoughts. @andrewsignori-aot @guru-aot @dheepak-aot

Reply:
what is the question here @ann-aot ? I didn't get.

Reply:
pls refer https://github.com/bcgov/SIMS/pull/1525#discussion_r1022268789 @dheepak-aot